kwave/utils/mapgen.py

```diff
@@ -2555,3 +2555,119 @@ def trim_cart_points(kgrid, points: np.ndarray):
     points = points[:, ind]
 
     return points
+
+
+import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Should these be moved to the start of the file?

Reply:
done